src/RobotsTxt.php

```diff
@@ -34,32 +34,77 @@ public static function create(string $source): self
 
     public function allows(string $url, ?string $userAgent = '*'): bool
```

Comment:
The remarks given above apply to this function too.

Reply:
Code changed. 

I don't know if it's a good idea to use null coalescent one false.